lib/driver.js

```diff
@@ -463,6 +459,24 @@ class EspressoDriver extends BaseDriver {
     await this.addDeviceInfoToCaps();
   }
 
+  async setWindowAnimationState(disableAnimation) {
```

Comment:
the argument name seems confusing to me. I would rather change it to `isEnabled`

Reply:
the default state for the variable is `true`.
`disableAnimation === true` animation is disabled
`disableAnimation === false` animation is enabled

`isEnabled === true` sounds like animation should be enabled but when the cap `disableWindowAnimation` value is true animation should be disabled
and wise versa for the `false` value of `disableWindowAnimation`.

Should I just change it to `animationState`. We are anyway setting the value true by `this.caps.disableWindowAnimation !== false` except for the explicit `false` value.

Reply:
it does not matter here the cap name is. It is about the arg name for this particular method. Eventually, `state` assumes multiple possible states, but this one could be on or off (true/false)

Reply:
> isEnabled === true sounds like animation should be enabled but when the cap disableWindowAnimation value is true animation should be disabled

Right, simply pass `disableWindowAnimation === false` to it (inverted condition)

Reply:
Done